src/ImageService.Broker/Consumers/RemoveImagesConsumer.cs

```diff
@@ -1,53 +1,53 @@
-﻿using LT.DigitalOffice.ImageService.Data.Interfaces;
+﻿using System.Threading.Tasks;
+using LT.DigitalOffice.ImageService.Data.Interfaces;
 using LT.DigitalOffice.Kernel.Broker;
 using LT.DigitalOffice.Models.Broker.Enums;
 using LT.DigitalOffice.Models.Broker.Requests.Image;
 using MassTransit;
-using System.Threading.Tasks;
 
 namespace LT.DigitalOffice.ImageService.Broker.Consumers
 {
-    public class RemoveImagesConsumer : IConsumer<IRemoveImagesRequest>
-    {
-        private readonly IImageMessageRepository _imageMessageRepository;
-        private readonly IImageNewsRepository _imageNewsRepository;
-        private readonly IImageProjectRepository _imageProjectRepository;
-        private readonly IImageUserRepository _imageUserRepository;
+  public class RemoveImagesConsumer : IConsumer<IRemoveImagesRequest>
+  {
+    private readonly IImageMessageRepository _imageMessageRepository;
+    private readonly IImageNewsRepository _imageNewsRepository;
```

Comment:
we work with news images throw controler

Reply:
done